src/ripple/protocol/TER.h

```diff
@@ -263,6 +263,8 @@ enum TECcodes : TERUnderlyingType
     tecINVARIANT_FAILED         = 147,
     tecEXPIRED                  = 148,
     tecDUPLICATE                = 149,
+    tecKILLED                   = 150,
+    tecNEGATIVE_BALANCE         = 151
```

Comment:
I am concerned with the proliferation of existing `tec` codes, considering that we currently allow only up to 159. The nice thing is that the limit isn't "hard" and we can update it, but we need to start thinking about it *now*.

Reply:
@nbougalis, I agree.  `tec` codes are the ones most likely to proliferate, since once one gets into the ledger we can never get rid of it.  And we only (initially) left ourselves room for 60 of them.

Not that this is a long term solution, but I just noticed that there is a gap in the assigned `tec` codes between 105 and 121.  Do you know anything about the history here?  Are those additional codes that we could use, or are they already consumed in some way?  Thanks.